wp-content/themes/core/assets/css/src/admin/block-editor/editor.pcss

```diff
@@ -19,18 +19,25 @@
 		margin-bottom: 0;
 	}
 
-	/* CASE: Alignment: Full */
-	.wp-block[data-align="full"] {
-		margin-left: auto;
-		margin-right: auto;
-	}
-
 	/* Reset/remove the browser's default margin n figure inside the block editor. */
 	figure {
 		margin: 0;
 	}
 }
 
+/**
+ * Reset left and right margin for full aligned blocks
+ * to overcome aggressive WordPress core styles.
+ */
+html :where(.editor-styles-wrapper) {
```

Comment:
Unrelated to columns work, but it seems like WordPress changed how they are selecting this so full-align blocks are being left justified as seen [in this screenshot.](http://p.tri.be/fNY1Rj)